packages/descendants/src/Highlight/README.md

```diff
@@ -0,0 +1,117 @@
+# Highlight Management
+
+A suite of utilities for managing the highlight state of Descendants.
```

Comment:
can you add some detail around how highlight state is similar to or different from active/focus/hover?